yarn-project/circuits.js/src/structs/avm/avm.ts

```diff
@@ -87,6 +87,7 @@ export class AvmExternalCallHint {
     returnData: Fr[],
     public readonly gasUsed: Gas,
     public readonly endSideEffectCounter: Fr,
+    public readonly bytecode: Buffer,
```

Comment:
Same as before. It would be good to have some doc on what this is right now and what it is supposed to be.